client/src/components/LogOutModal/LogOutModal.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { AlertModal } from '..';
+import { toggleIsModalOpen, toggleIsLogin } from '../../store/slices/userSlice/userSlice';
+
+const LogOutModal = () => {
+  const { isModalOpen } = useSelector((store) => store.userReducer);
+
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+
+  const closeModal = () => {
+    dispatch(toggleIsModalOpen());
+  };
+
+  const logOut = () => {
+    localStorage.removeItem('token');
+    dispatch(toggleIsLogin());
```

Comment:
```suggestion
    dispatch(toggleIsLogin());
    dispatch(toggleIsModalOpen());
```

you forgot to dispatch the action for closing the modal after confirming logging out

Reply:
done